src/BlazorSignalR.JS/src/WebSocketsTransport.ts

```diff
@@ -21,25 +21,25 @@ export class WebSocketsTransport {
         }
 
         webSocket.onopen = (_event: Event) => {
-            managedObj.invokeMethod<void>("HandleWebSocketOpened");
+            managedObj.invokeMethodAsync<void>("HandleWebSocketOpened");
```

Comment:
These should probably be awaited? To ensure errors etc are not lost.

Reply:
This is done. The complete callback is an async function now.